src/services/WorkPeriodService.js

```diff
@@ -519,10 +519,11 @@ async function searchWorkPeriods (currentUser, criteria, options = { returnAll:
       required: false
     }]
   }
-  const workPeriods = await WorkPeriod.findAll(queryCriteria)
+  const workPeriods = await WorkPeriod.findAndCountAll(queryCriteria)
+  const total = await WorkPeriod.count({ where: filter })
```

Comment:
@xxcxy I didn't know such a method exists. I wonder why you didn't use the output from this method and call `await WorkPeriod.count({ where: filter })` anyway below?

Can we just call `findAndCountAll`?

Reply:
Strange, I forgot to change this. I tried findAndCountAll, the count returned by this method is wrong.

Reply:
@maxceem @xxcxy
Fyi, `findAndCountAll` is safe when we don't have `include` option in query. Otherwise it gives wrong count number and becomes complex to solve.

Reply:
Thanks @eisbilir. I think than it’s better not use it, because if we use it one place then new developer can copy paste it to a new place with include and would not notice that it works incorrectly there.